src/fides/api/models/privacy_request.py

```diff
@@ -523,14 +524,6 @@ def verify_identity(self, db: Session, provided_code: str) -> "PrivacyRequest":
         self.save(db)
         return self
 
-    def cache_task_id(self, task_id: str) -> None:
-        """Sets a task_id for this privacy request's asynchronous execution."""
-        cache: FidesopsRedis = get_cache()
-        cache.set(
-            get_async_task_tracking_cache_key(self.id),
-            task_id,
-        )
-
```

Comment:
This was an unused method, instead we cached this id without having the privacy request in memory from the request runner service.